packages/odyssey-react/src/components/CircularLoadIndicator/CircularLoadIndicator.tsx

```diff
@@ -0,0 +1,90 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import type { ComponentPropsWithRef } from "react";
+import { forwardRef } from "react";
+import { useOmit, withStyles } from "../../utils";
+import styles from "./CircularLoadIndicator.module.scss";
+
+export interface CircularLoadIndicatorProps
+  extends Omit<
+    ComponentPropsWithRef<"span">,
+    "style" | "className" | "role" | "aria-valuemin" | "aria-valuemax"
+  > {
+  /**
+   * id used as reference by loading region
+   */
+  id: string;
```

Comment:
hmmm, is the use case here `aria-describedby` directed at the `id` owned by our component here? This is the accessibility pattern I have in mind for form elements disabled for load etc. I think we can add a sensible default to improve the DX by having `id` auto generated via `oid`. It will always be accessible on the `ref` for the consumer this way they are not in charge of having to generate a unique ID themselves for a common workflow but can also customize it (via oid argument) if necessary!